### PR TITLE
Fix the link to TxServiceModel

### PR DESCRIPTION
**Pull request by katspaugh**
The link was broken, I've replaced it with a relative URL.

Also added a link to the JSON RPC docs in the Ethereum wiki.

### CODE REVIEW

**Comment by katspaugh** [comment]
I have read the CLA Document and I hereby sign the CLA

File: packages/safe-apps-sdk/README.md [hunk 1]
```diff
@@ -114,10 +114,12 @@ Once you received safe transaction hash, you might want to get the status of the
 const tx = await sdk.txs.getBySafeTxHash(safeTxHash);
 ```
 
-It will return the following structure https://github.com/gnosis/safe-apps-sdk/blob/development/src/types.ts#L182 or throw an error if the backend hasn't synced the transaction yet
+It will return a [TxServiceModel](src/types.ts#L164-L194) structure or throw an error if the backend hasn't synced the transaction yet
 
 ## RPC Calls
 
+The SDK exposes Ethereum's [JSON-RPC methods](https://eth.wiki/json-rpc/API).
```
Comment (mikheevm) [inline packages/safe-apps-sdk/README.md:121]: I'm not sure whether we should mention that we only expose read methods, not the whole API 🤔

**Comment by mikheevm** [comment]
Thanks! I added you to the CLA's action allowlist: https://github.com/gnosis/safe-apps-sdk/pull/131/commits/8ed91a08091f681a917354d8a24717ccd3e6db22

Feel free to do the same for other safe repos 🙂

File: packages/safe-apps-sdk/README.md (hunk 1 above)
Comment (katspaugh) [reply packages/safe-apps-sdk/README.md:121]: I'll adjust it 👌

**Comment by katspaugh** [comment]
> Thanks! I added you to the CLA's action allowlist: 8ed91a0
Thank you, will do 👍

**Review by mikheevm — APPROVED** [review]

**Pull request merged**